tests/test_utils.py

```diff
@@ -240,9 +258,13 @@ def samplesheet_file_from_runfolder(runfolder):
 Adapter,,,,,,,,
 AdapterRead2,,,,,,,,
 ,,,,,,,,
-[Data],,,,,,,,
+
 """
-    samplesheet_data = samplesheet_data_for_runfolder(runfolder)
+    if demultiplexer == "bcl2fastq":
+        header_stuff = header_stuff + "[Data],,,,,,,,\n"
```

Comment:
It would be nice if the `SAMPLESHEET_DATA_HEADERS_DICT` was imported and used here. So that we only need to update them in one place.